tests/src/unit-readme.cpp

```diff
@@ -158,10 +158,10 @@ TEST_CASE("README" * doctest::skip())
             CHECK(foo == true);
 
             // other stuff
-            j.size();     // 3 entries
-            j.empty();    // false
-            j.type();     // json::value_t::array
-            j.clear();    // the array is empty again
+            CHECK(j.size() == 3);                     // 3 entries
```

Comment:
Just FYI: I copied the code snippets from the README here to make sure they compile in the first place. In an ideal world, the README would be shorter and have more pointers to the documentation - or we'd have a means to strip code snippets from the README and check them automatically.

Anyway, I appreciate this approach of converting the README code to proper tests.

Reply:
I added the first 3 checks just because of lgtm. It's not really necessary, since lgtm ignore test code, but it seemed like a good idea.